src/commonControls/questionnaireControl/QuestionnaireControlBuiltIns.ts

```diff
@@ -269,10 +292,11 @@ export namespace QuestionnaireControlAPLPropsBuiltIns {
                     type: 'Container',
                     width: '100vw',
                     height: '100vh',
+                    disabled: '${disableContent}',
                     bind: [
                         {
                             name: 'disableContent',
-                            value: false,
+                            value: '${wrapper.disableScreen}',
```

Comment:
`wrapper.disableScreen` ?? I couldn't find it in the return payload of `questionnaireDataSourceGenerator`

Reply:
It was dead code. Removed!